lib/matplotlib/backends/backend_qt.py

```diff
@@ -160,46 +160,6 @@ def _create_qApp():
     return app
 
 
-def _allow_super_init(__init__):
```

Comment:
Do we want to leave this decorator in place even if we do not need it in case anyone is doing further MI on our classes?

Reply:
It's not a generic solution (but rather specialized for pyside), it's private, and third parties can always vendor the code (or also try whether swapping parent order helps...).